From ff1720307554405a422281591dba9b79f085799b Mon Sep 17 00:00:00 2001
From: Yi EungJun <semtlenori@gmail.com>
Date: Tue, 31 Jan 2017 00:24:06 +0900
Subject: [PATCH 55/59] journal-gatewayd: return -EINVAL if ARG_TRUST and
 HAVE_GNUTLS (#5181)

This bug was introduced by 1aa1e59.
(cherry picked from commit 3ab2feefd7d1918e8f0d0edaea46a87cbed0f211)
---
 src/journal-remote/journal-gatewayd.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/journal-remote/journal-gatewayd.c b/src/journal-remote/journal-gatewayd.c
index be6d41c..c7c6224 100644
--- a/src/journal-remote/journal-gatewayd.c
+++ b/src/journal-remote/journal-gatewayd.c
@@ -958,6 +958,7 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 #else
                         log_error("Option --trust is not available.");
+                        return -EINVAL;
 #endif
                 case 'D':
                         arg_directory = optarg;
-- 
2.8.3

